test/functional/users_controller_test.rb

```diff
@@ -322,6 +322,7 @@ class UsersControllerTest < ActionController::TestCase
   end
 
   test "#login renders login page" do
+    session.clear
```

Comment:
Please disregard, I don't think it's worthwhile any more - there were more instances of it in the previous version of the PR.